REFERENCE.md

```diff
@@ -267,7 +277,7 @@ If you observe poor performance on a dataset with a small number of examples, th
 
 ### The variance estimates are jumpy or very large.
 
-In this case, it would be good to try growing a larger number of trees. Obtaining good variance estimates often requires growing more trees than it takes to only obtain accurate predictions.
+In this case, it would be good to try growing a larger number of trees. Obtaining good variance estimates often requires growing more trees than it takes to only obtain accurate predictions. See the discussion in [`num.trees`](#num-trees) and [Merging forests](#merging-forests).
```

Comment:
This is a nice idea to use hyperlinks to the different sections, there are a bunch of other references in this text that could have been hyperlinks as well. Although 'Merging forests' seems to work, I don't know if the `num-trees` reference is correct -- it doesn't show up as a link in the 'rich diff'.